Chap_API_Struct.tex

```diff
@@ -4776,12 +4776,20 @@ \subsection{Query attributes}
 
 %
 \declareAttributeNEW{PMIX_QUERY_NUM_PSETS}{"pmix.qry.psetnum"}{size_t}{
-Return the number of psets defined in the specified range (defaults to session).
+Return the number of process sets defined in the specified range (defaults
+to session).
 }
 
 %
 \declareAttributeNEW{PMIX_QUERY_PSET_NAMES}{"pmix.qry.psets"}{char*}{
-Return a comma-delimited list of the names of the psets defined in the specified range (defaults to session).
+Return a comma-delimited list of the names of the process sets defined in the
+specified range (defaults to session).
+}
+
+%
+\declareAttributeNEW{PMIX_QUERY_PSET_MEMBERSHIP}{"pmix.qry.pmems"}{pmix_proc_t*}{
+Return an array of \refstruct{pmix_proc_t} of the members of the specified
+process set.
 }
```

Comment:
Per my previous comment, I suggest that we create a parallel set of attributes for process groups (`PMIX_QUERY_NUM_PGROUPS`, `PMIX_QUERY_PGROUP_NAMES`, `PMIX_QUERY_PGROUP_MEMBERSHIP`) instead of combining them.